### DISCUSSION TITLE
2023년 06월 26일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#60] [General]
# 2023년 06월 26일

LOG 동생이 입대했다. 전날에 같이 술 마신 것 때문에 조금 속이 안 좋았다..ㅋㅋ 점심은 동생이 먹고 싶어했던 육회 비빔밥을 먹으러 익산에 갔다. 비빔밥 집인데 사람이 굉장히 많길래 뭘까 했는데 백종원의 3대천왕에 나왔던 곳이었다. 꽤나 맛있었다…

https://vero.wiki/2023-06-26/

**Comment by drunkenhw** [comment]
요즘 군대는 군대도 아니래요~

**Reply by Cyma-s** [reply]
장유유서
